### DISCUSSION TITLE
Keda Not able to Scale job based on Azure queue message

### DISCUSSION THREAD

**Discussion by sachinagarwal010** [kedacore/keda#3340] [Q&A / Need Help]
We have Azure Storage Queue having few messages there and Keda is creating the scaled Job based on Azure queue message available (Auto Scaling max Replica Set was set to 30) but when we already have scaled 30 jobs and those jobs are finished then keda started picking the azure queue messages 1 by 1 not based on the Auto Scaling max Replica Set was set to 30.

**Comment by tomkerkhove** [comment]
Hm I'm not sure I'm getting what you are saying - So you're saying it's scaling out to the max and then there is an issue or?
